code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/MessagingConstants.java

```diff
@@ -407,4 +407,26 @@ public class PayloadKeys {
             private PayloadKeys() {}
         }
     }
+
+    public final class UIKeys {
+        public static final String CONTENT = "content";
+        public static final String URL = "url";
+        public static final String DARK_URL = "darkUrl";
+        public static final String INTERACT_ID = "interactId";
+        public static final String ACTION_URL = "actionUrl";
+        public static final String TEXT = "text";
+        public static final String STYLE = "style";
+        public static final String ID = "id";
+        public static final String TITLE = "title";
+        public static final String BODY = "body";
+        public static final String IMAGE = "image";
+        public static final String BUTTONS = "buttons";
+        public static final String DISMISS_BTN = "dismissBtn";
+
+        public class DismissButtonStyle {
```

Comment:
Can we keep the class hierarchy consistent? If we plan to have everything inside `UIKeys` lets remove this. If not, lets make sure all keys are under respective classes like `Text`, `Button` and `Image`